hr_hr.lang

```diff
@@ -1,215 +1,349 @@
-gui.xaero_entity_depth=Prikaži dubinu entityja
-gui.xaero_show_full_amount=Puna količina itema u ruci
-gui.xaero_show_flowers=Prikaži cvijeće
-gui.xaero_show_entity_model=Prikaži model entityja
-gui.xaero_waypoint_names=Ime PT preko udaljenosti
-gui.xaero_waypoints_distance=Max. udaljenost prikaza PT
-gui.xaero_teleport_command=Naredba za teleport do PT
-gui.xaero_waypoints_distance_min=Min. udaljenost prikaza PT
-gui.xaero_other_teams=Ostali timovi
-gui.xaero_entity_info_armour_numbers=Oklop u brojevima
-gui.xaero_entity_info_armour=Vrijednost oklopa
\ No newline at end of file
+gui.xaero_confirm=Potvrdi
+gui.xaero_cancel=Poništi
+gui.xaero_back=Natrag
+gui.xaero_delete=Obriši
+gui.xaero_OK=U redu
+gui.xaero_add_edit=Dodaj/Uredi
+gui.xaero_new_waypoint=Nova putna točka
+gui.xaero_initial=initial
+gui.xaero_edit_waypoint=Uredi putnu točku
+gui.xaero_armour_status_settings=Postavke stanja oklopa
+gui.xaero_choose_a_preset=Odaberi predložak
+gui.xaero_choose_an_interface=Odaberi sučelje
+gui.xaero_armour_status=Stanje oklopa
+gui.xaero_notifications=Obavijesti
+gui.xaero_minimap=Mini karta
+gui.xaero_miscellaneous=Razno
+gui.xaero_edit_mode=Način uređivanja
+gui.xaero_instructions=Upute/Pomoć
+gui.xaero_not_ingame=Moraš biti u igri za korištenje načina uređivanja!
+gui.xaero_welcome_edit_mode=Dobrodošli u način uređivanja!
+gui.xaero_howto_select=Lijevi klik na nekom od sučelja za označavanje.
+gui.xaero_howto_drag=Lijevi klik i povlačenje na odabranom sučelju za pomicanje tog sučelja.
+gui.xaero_howto_deselect=Desni klik za odznačavanje odabranog sučelja.
+gui.xaero_howto_center=Pritisni "C" kako bi centrirao i decentrirao sučelje
+gui.xaero_different_centered=Neka sučelja imaju drukčiji centrirani izgled.
+gui.xaero_howto_flip=Pritisni "F" da okreneš ili vratiš označeno sučelje.
+gui.xaero_howto_settings=Pritisni "S" za brzi pristup postavkama označenog sučelja.
+gui.xaero_howto_preset=Odaberi predložak klikom na gumb "Odaberi predložak".
+gui.xaero_howto_save=Spremi promjene klikom na gumb "Potvrdi".
+gui.xaero_howto_cancel=Poništi promjene klikom na gumb "Poništi".
+gui.xaero_minimap_settings=Postavke mini karte
+gui.xaero_miscellaneous_settings=Razne postavke
+gui.xaero_notification_settings=Postavke obavijesti
+gui.xaero_better_pvp_settings=Better PVP postavke
+gui.xaero_server_disabled=Poslužitelj je možda onemogućio neke mogućnosti moda.
+gui.xaero_waypoints=Putne točke
+gui.xaero_box_potion_effects=Sučelje koje prikazuje trenutno aktivne efekte napitaka.
+gui.xaero_box_armour_status=Sučelje koje prikazuje oklop koji trenutno nosiš i predmet u ruci.
+gui.xaero_box_sprint=Trčanje držanjem samo jedne tipke (podesivo).
+gui.xaero_box_sneak=Omogućuje uključivanje/isključivanje šuljanja i stajanje šuljajući se bez potrebe držanja tipke.
+gui.xaero_box_notifications=Obavijesti na zaslonu koje prikazuju različite stvari kao što su: niska glad, nisko zdravlje, eksplozija u blizini ili kad si pogođen strijelom/ama.

```

Comment:
As I later explained, this was done because correct translation sounds like something from some of Shakespeare's books, aka very weird. You can choose from some of those I typed down there or we can revert it to the old translation.

Reply:
I'm not saying that you should revert to the old translation, but your translation just doesn't seem correct. Is it possible to fix it without sounding weird?